recipes/libconda/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "4.0.0" %}
+{%set sha256 = "8cc07dc407527fdb4d028c8caf67757f6df08d045aa03e1ecd9796aa0489a3a2" %}
+
+package:
+  name: libconda
+  version: {{ version }}
+
+source:
+  fn: libconda-{{ version }}.tar.gz
+  url: https://github.com/ilanschnell/libconda/archive/{{ version }}.tar.gz
```

Comment:
Now I remember why I did this. There is no tag on the org repo. I have added an issue ( https://github.com/conda/libconda/issues/1 ) upstream for this change.